### PR TITLE
mark acceptance tests as @smokeTest

### PR DESCRIPTION
**Pull request by individual-it**
## Description
mark some tests as `@smokeTests`

**goals:**
1. run tests that check if the user can generally achieve what is written in the feature description "As ... I want to ... So that ..."
1. make sure that ownCloud works correctly on top of a specific environment

**rules by which I choose the tests:**
1. only shallow tests
1. if applicable test all parts of CRUD
1. touch every part of the API / every endpoint
1. check all routes
1. test only the happy-path, except if the main reason for the feature is to block some action e.g. quota
1. only test very basic security features
1. for negative settings only test the negative outcome, no need to check if the setting switches off something else
1. don't test edge-cases and combinations of features (every action with sharing, subfolders, etc.) except if that would test a specific environment (see goal 2.). This specific environment tests should:
    1. check for UTF / i18n in database
    1. check for UTF / i18n in filesystem
    1. check that the correct UTF string was delivered to the browser (no need to test the rendering in browser e.g. due to missing fonts)
    1. check that file operations really happen (was a file moved / overwritten / deleted?)
    1. check if routes with and without `index.php` work correctly, depending on the environment (no test to check that systematically yet)
    1. check sending of emails
    1. check that correct HTTP headers are send

all other tests are still valid and useful, but just not smoke tests

## Issue
https://github.com/owncloud/QA/issues/583

## Motivation and Context
run smoke tests during the build process

## How Has This Been Tested?
reviewers have to test if my "rules" and choices are reasonable

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Comment by patrickjahns** [comment]
Drone is 🔴 :'(

**Comment by individual-it** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

File: tests/acceptance/features/apiMain/status.feature [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @api
 Feature: Status
 
+  @smokeTest
   Scenario: Status.php is correct
```
Comment (patrickjahns) [inline tests/acceptance/features/apiMain/status.feature]: We might be testing customer docker containers with a specific theme enabled. Themes can override the strings in status.php. Not sure if we can keep it like this.

Also the test for the customer container might require the values to be product specific

File: tests/acceptance/features/apiMetadataApps/tags.feature [hunk 2]
```diff
@@ -9,11 +10,11 @@ Feature: tags
       |<tag_name>|normal|
     And the following tags should exist for "user0"
       |<tag_name>|normal|
-
     Examples:
       | tag_name            |
       | JustARegularTagName |
       | 😀                  |
+             |सिमप्ले                   |
```
Comment (patrickjahns) [inline tests/acceptance/features/apiMetadataApps/tags.feature]: not smoke test related - maybe different PR ?

File: tests/acceptance/features/apiMetadataApps/tags.feature [hunk 3]
```diff
@@ -39,17 +41,18 @@ Feature: tags
...
     And user "admin" has created a "normal" tag with name "<tag_name>"
     When user "user0" edits the tag with name "<tag_name>" and sets its name to "AnotherTagName" using the WebDAV API
     Then the following tags should exist for "admin"
       | AnotherTagName | normal |
-
     Examples:
       | tag_name            |
       | JustARegularTagName |
       | 😀                  |
+             |सिमप्ले                   |
```
Comment (patrickjahns) [inline tests/acceptance/features/apiMetadataApps/tags.feature]: not smoke test related - maybe different PR ?

File: tests/acceptance/features/apiProvisioning-v1/addGroup.feature [hunk 4]
```diff
@@ -17,8 +29,6 @@ So that I can more easily manage access to resources by groups rather than indiv
 			| new-group           | dash                                    |
 			| the.group           | dot                                     |
 			| left,right          | comma                                   |
-			| España              | special European characters             |
-			| नेपाली                                   | Unicode group name                       |
```
Comment (patrickjahns) [inline tests/acceptance/features/apiProvisioning-v1/addGroup.feature]: don't quite understand the changes here - please explain

File: tests/acceptance/features/apiProvisioning-v1/addToGroup.feature [hunk 5]
```diff
@@ -19,8 +33,6 @@ So that I can give a user access to the resources of the group
 			| new-group           | dash                                    |
 			| the.group           | dot                                     |
 			| left,right          | comma                                   |
-			| España              | special European characters             |
-			| नेपाली              | Unicode group name                      |
```
Comment (patrickjahns) [inline tests/acceptance/features/apiProvisioning-v1/addToGroup.feature]: also don't quite understand changes here - please explain

File: tests/acceptance/features/apiMetadataApps/tags.feature (hunk 2 above)
Comment (individual-it) [reply tests/acceptance/features/apiMetadataApps/tags.feature]: its related as we wanted to have some UTF test, but there was no UTF test yet

File: tests/acceptance/features/apiProvisioning-v1/addGroup.feature (hunk 4 above)
Comment (individual-it) [reply tests/acceptance/features/apiProvisioning-v1/addGroup.feature]: moved a simple test and UTF related tests into the smoketest section, so no need to have them doubled up

**Comment by patrickjahns** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase failed! Please rebase your pull request manually via the command line.

Reason:
```
Command: git rebase base/master

Exit Code: 128

Cause:

.git/rebase-apply/patch:814: trailing whitespace.
		And the HTTP status code should be "200" 
.git/rebase-apply/patch:1267: trailing whitespace.
		And the HTTP status code should be "200" 
.git/rebase-apply/patch:2071: trailing whitespace.
	@smokeTest
.git/rebase-apply/patch:2080: trailing whitespace.

.git/rebase-apply/patch:2081: trailing whitespace.
	@smokeTest
warning: squelched 5 whitespace errors
warning: 10 lines add whitespace errors.
error: Failed to merge in the changes.


```

**Comment by PVince81** [comment]
proposed rules make sense

**Comment by individual-it** [comment]
@patrickjahns resolved conflicts and deleted the tags on status.php and federation tests
anything else or merging?

**Comment by patrickjahns** [comment]
Let's merge as soon as ✅ - we can add/remove more tests later on

File: tests/acceptance/features/webUIRestrictSharing/disableSharing.feature [hunk 6]
```diff
@@ -10,6 +10,7 @@ So that users cannot share files
 		|user1   |1234    |User One   |u1@oc.com.np|
 		
 	@TestAlsoOnExternalUserBackend
+	@smokeTest
```
Comment (phil-davis) [inline tests/acceptance/features/webUIRestrictSharing/disableSharing.feature]: Does having the tags on multiple lines work?
@individual-it

Comment (phil-davis) [reply tests/acceptance/features/webUIRestrictSharing/disableSharing.feature]: I  tried it - yes, it works having tags on multiple lines.

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
backport in  #32478
